src/views/virtualmachines/actions/components/CloneVMModal/components/CloningStatus.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, useEffect } from 'react';
+
+import { modelToGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
+import VirtualMachineCloneModel from '@kubevirt-ui/kubevirt-api/console/models/VirtualMachineCloneModel';
+import { V1alpha1VirtualMachineClone } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Alert,
+  AlertVariant,
+  ProgressStep,
+  ProgressStepper,
+  ProgressStepVariant,
+  Stack,
+} from '@patternfly/react-core';
+
+import { CLONING_STATUSES, STATUS_TO_PROGRESS_VARIANT } from '../utils/constants';
+
+type CloningStatusProps = {
+  onComplete: (status: CLONING_STATUSES) => void;
+  vmCloneRequest: V1alpha1VirtualMachineClone;
+};
+
+const CloningStatus: FC<CloningStatusProps> = ({ onComplete, vmCloneRequest }) => {
+  const { t } = useKubevirtTranslation();
+
+  const [freshVMCloneRequest] = useK8sWatchResource<V1alpha1VirtualMachineClone>(
+    vmCloneRequest && {
```

Comment:
i dont understand why u need a new clone object is vmclonerequest is a clone object

Reply:
The vmCloneRequest status does not get updated during time

Reply:
At the beginning, the freshCloneRequest is undefined so the ui will show the status of vmCloneRequest. When the freshCloneRequest is on watch the status component will be re-rendered on status change and the progress will show the actual status